src/mainwindow.cpp

```diff
@@ -74,7 +74,12 @@ void MainWindow::keyPressEvent(QKeyEvent *event)
     }
     return QWidget::keyPressEvent(event);
 }
-
+void MainWindow::moveEvent(QMoveEvent *event)
+{
+    Q_UNUSED(event);
+    const auto global = mapToGlobal(rect().center());
+    mp_localKiwixServer->move(global.x() - mp_localKiwixServer->width() / 2, global.y() - mp_localKiwixServer->height() / 2);
+}
```

Comment:
I'm not sure about that.
It is to the windows manager to handle this.
Some WM will move the modal dialog, some don't, be we must not enforce this.
(I personally deactivate this feature all the time. I really dislike to have a dialog windows hiding what I want to see)

Reply:
I manually center the widget because @kelson42 wants a frameless dialog. One can't drag a frameless window, so I think I'd center it. As for the modal, that's also requested by kelson.

Reply:
OK to remove the centering. Let leave this to the Windows Manager.

Reply:
done